ray-on-gke/tpu/kuberay-tpu-webhook/deployments/validating-webhook-cfg.yaml

```diff
@@ -5,30 +5,6 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: ray-system/kuberay-tpu-webhook-certs
 webhooks:
-  - name: pods-kuberay-tpu-webhook.ray-system.svc
-    admissionReviewVersions: [v1]
-    sideEffects: NoneOnDryRun
-    namespaceSelector:
-      matchExpressions:
-      - key: kubernetes.io/metadata.name
-        operator: NotIn
-        values:
-        - kube-system
-        - kube-node-lease
-    clientConfig:
-      service:
-        name: kuberay-tpu-webhook
-        namespace: ray-system
-        path: /validate
-    rules:
-      - operations: ["DELETE"]
```

Comment:
What's the reason for no longer intercepting DELETE?

Reply:
Since we check the current Pod state of the RayCluster with the k8s client each time we get a Pod creation request, we no longer need to keep track of the Pods that get deleted. The validation step for deleting Pods was previously just used to update the in-memory mapping.